bin/orphaned-prs

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+# This script checks for orphaned PR environments by listing all PR workspaces
+# and checking if the associated PR is closed. If the PR is closed the
+# resources in the workspace should have been destroyed and the workspace
+# deleted, so existing workspaces for closed PRs are considered orphaned.
+# -----------------------------------------------------------------------------
+set -euo pipefail
+
+app_name="$1"
+
+echo "::group::Initialize Terraform"
+echo terraform -chdir="infra/${app_name}/service" init -input=false -reconfigure -backend-config="dev.s3.tfbackend"
+terraform -chdir="infra/${app_name}/service" init -input=false -reconfigure -backend-config="dev.s3.tfbackend"
+echo "::endgroup::"
+
+echo "::group::List PRs with PR environments"
```

Comment:
```suggestion
echo "::group::PR environments, which corresponds to a list of PRs"
```

Reply:
The suggestion doesn't seem accurate. What we are listing are PR numbers for PRs that have an active PR environment.